packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -36,11 +36,11 @@ import { useCfdStore } from '../Stores/Modules/CFD/Helpers/useCfdStores';
 import { CFD_PLATFORMS, CATEGORY } from '../Helpers/cfd-config';
 import classNames from 'classnames';
 import { getDxCompanies, getMtCompanies, TDxCompanies, TMtCompanies } from '../Stores/Modules/CFD/Helpers/cfd-config';
-import '../sass/cfd.scss';
+import './mt5-create-password/mt5-create-password.scss';
```

Comment:
why do we need to import mt5-create-password.scss to cfd-password-modal? can't we import it only to mt5-create-password.tsx?

Reply:
some elements are using the styles provided by mt5-create-password.scss.

Reply:
before removing import:
![Screenshot 2024-08-26 at 1 37 57 PM](https://github.com/user-attachments/assets/b8ca30ae-9179-4c9b-8d8c-3a329f3d4fe4)

after removing import: 
![Screenshot 2024-08-26 at 1 39 42 PM](https://github.com/user-attachments/assets/1ae8a57b-6db1-4e2b-a75c-d9e0acb53ccb)